packages/image_picker/image_picker/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -255,6 +302,11 @@
 						ProvisioningStyle = Automatic;
 						TestTargetID = 97C146ED1CF9000F007C117D;
 					};
+					6801C8352555D726009DAF8D = {
```

Comment:
This is metadata about the test target, so which version of Xcode was used to create it, and how how the product should be signed.